ownCloud/Tools/VendorServices.swift

```diff
@@ -53,13 +66,20 @@ class VendorServices : NSObject {
 	func recommendToFriend(from viewController: UIViewController) {
 		let message = """
 <p>I want to invite you to use ownCloud on your smartphone!</p>
-<a href="https://itunes.apple.com/app/owncloud/id543672169?mt=8">Download here</a>
+<a href="https://itunes.apple.com/app/id1359583808?mt=8">Download here</a>
```

Comment:
@pablocarmu we could make an AppStore link and also E-Mail addresses configurable trough build settings rather then hard-coding them. What if a customer would like to release branded AppStore version? cc: @michaelstingl what do you think?

Reply:
I can code this using the `OCClassSettings` mechanism, so we can provide default email & AppStore link that could be overwritten by an MDM or config file.

Reply:
Yes, we need a boolean switch to disable it, and another parameter (URL?) that can be customised.

Reply:
These are the new variables I've coded:

- **Variables**:
	- app-store-link : Points to the app's link in the app store.
		- type: String
		- default: `https://itunes.apple.com/app/id1359583808?mt=8`
	- feedback-email : Email to send the feedback mail.
		- type: String
		- default: `ios-app@owncloud.com`
	- recommend-to-friend-enabled : Option to send en email with the App Store link.
		- type: Bool
		- default: `true`
	- send-feedback-enabled : Send an email to feedback-email with some feedback.
		- type: Bool
		-default: `true`